colossalai/zero/low_level/low_level_optim.py

```diff
@@ -506,3 +469,47 @@ def no_sync(self):
             yield
         finally:
             self.require_grad_sync = old_require_grad_sync
+
+
+class LowLevelZeroOptimizer(TPLowLevelZeroOptimizer):
```

Comment:
If this is like a complete inheritance, why not just make it the same class?

Reply:
In fact, there is a tiny difference.
`LowLevelZeroOptimizer` would not receive the kwarg `tp_process_group`, while `TPLowLevelZeroOptimizer` would.
The reason is quite simple: passing `tp_process_group` for naive ZeRO is quite weird.

Reply:
Let me make it more clear:
For naive ZeRO, it's weird that you can pass a `tp_process_group` to it because in common sense, tp has nothing to do with ZeRO.
However, passing a `tp_process_group` to a class named "TP" and "ZeRO" seems to be much more reasonable.

I would write a README for the usage.

Reply:
I mean you can make tp_process_group default to None in the `LowLevelZeroOptimizer`?

Reply:
You are right.
The two classes may introduce unnecessary workloads for users as well.
I'll change it.